dev/postgres-initdb/init-extensions.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Do we need this file? why?

Reply:
Because the function `uuid_generate_v4()` assumes the extension exists on the database. When I tried to start the services, it threw an error because the function didn't exists.

Reply:
understood